INSTALL.md

```diff
@@ -22,11 +23,11 @@ meson setup build -Dforce_fallback_for=
 ```
 
 Depending on the build configuration, HSE has the following additional
-dependencies for various internal tools:
+dependencies for various internal tools and documentation:
 
-* [mongo-c-driver](https://github.com/mongodb/mongo-c-driver)
-* [ncurses](https://invisible-island.net/ncurses/announce.html)
-* [HdrHistogram_c](https://github.com/HdrHistogram/HdrHistogram_c)
+* [mongo-c-driver](https://github.com/mongodb/mongo-c-driver) `>= 1.17.3`
+* [ncurses](https://invisible-island.net/ncurses/announce.html) `>= 6.1.20180127`
+* [HdrHistogram_c](https://github.com/HdrHistogram/HdrHistogram_c) `>= 0.11.2`
```

Comment:
I will get rid of the need for ncurses at some point, one less dependency..

Reply:
Works for me :).